src/coreclr/scripts/genEventPipe.py

```diff
@@ -72,26 +72,6 @@ def generateMethodSignatureWrite(eventName, template, extern, runtimeFlavor):
     sig_pieces.append(")")
     return ''.join(sig_pieces)
 
-def includeProvider(providerName, runtimeFlavor):
-    if runtimeFlavor.coreclr and providerName == "Microsoft-DotNETRuntimeMonoProfiler":
-        return False
-    else:
-        return True
-
-def includeEvent(inclusionList, providerName, eventName):
```

Comment:
Isn't this method still called in this source, generateClrEventPipeWriteEventsImpl?

Reply:
Its now moved to genEventing.py and imported for this script.